### ISSUE TITLE
Incorrect use of bash variable

### ISSUE THREAD

**Issue by jpagex** [PostgREST/postgrest#1796]
Hello, thank you for this great project and for your fantastic work!

I came across this line while reading the code. I think the `tmpdir` is missing a `$` to be correctly handled as a variable. Is that correct?

https://github.com/PostgREST/postgrest/blob/9c79a4174c62cbdea057738df2e4de0598349e33/test/with_tmp_db#L43

**Comment by monacoremo** [comment]
Absolutely, it should! Great catch, thank you for taking such a detailed look into the dev tooling!

Happy to merge your PR if you'd like to submit one.

**Comment by jpagex** [comment]
Thank you for your fast response. I just created the PR. Let me know if this is OK.
